[Fix docs] Add Azure supported images (#1780)
epiphany-platform/epiphany: docs/home/howto/CLUSTER.md

    epicli apply -f newcluster.yml
    ```

> ## Note for Red Hat / CentOS Azure images

Reviewer: 3 times # if this is a subpart of "How to create an Epiphany cluster on a cloud provider"

Author: Added subpart

Depending on your requirements you may need to create cluster based on Red Hat family (RHEL/CentOS) system. Be aware that Epiphany supports only images with RAW partitioning, and attached to standard RHEL repositories. Please read this article for more details:

https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-images#rhel-7-image-types

It means that actually for Azure Cloud Epiphany supports Red Hat 7.6 - 7.7 and CentOS 7.6 - 7.8 Versions (with RAW partitioning and attached to RHEL repositories).
List of supported images will be extended in next releases.

Example config for Red Hat:

> ```

Reviewer: Use:
'''yaml

Reviewer: Also maybe make sure that its know that these changes need to be made in the infrastructure/virtual-machine documents.

Author: Added

specification:
  storage_image_reference:
    publisher: RedHat
    offer: RHEL
    sku: 7-RAW
    version: "7.7.2019090418"
```
Example config for CentOS:

> ```

Reviewer: Use:
'''yaml

Author: Added yaml and "kind" part

    epicli apply -f newcluster.yml
    ```

### Note for Red Hat / CentOS Azure images

> Depending on your requirements you may need to create cluster based on Red Hat family (RHEL/CentOS) system. Be aware that Epiphany supports only images with RAW partitioning, and attached to standard RHEL repositories. Please read this article for more details:
> 
> https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-images#rhel-7-image-types

Reviewer 2: ```suggestion
For RHEL and CentOS, Epiphany currently supports only images with RAW partitioning and attached to standard RHEL repositories. For more details, refer to [Azure documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/redhat/redhat-images#rhel-7-image-types).
```

> It means that actually for Azure Cloud Epiphany supports Red Hat 7.6 - 7.7 and CentOS 7.6 - 7.8 Versions (with RAW partitioning and attached to RHEL repositories).

Reviewer 2: ```suggestion
It means that actually for Azure Cloud Epiphany supports RHEL 7.6 - 7.7 and CentOS 7.6 - 7.8 versions (with RAW partitioning and attached to standard repositories).
```

List of supported images will be extended in next releases.

> Example config for Red Hat:

Reviewer 2: ```suggestion
Example config for RHEL:
```
